ci-operator/config/openstack-k8s-operators/keystone-operator/openstack-k8s-operators-keystone-operator-ds_test.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: keystone-ds-test
+  cluster_claim:
```

Comment:
You can remove cluster_claim, since you don't want to test against upstream Hive.
This will only delay your job if Hive doesn't have a cluster ready for you.

Reply:
I'll this file got added by mistake. sorry , 'llI remove it 
my intention was only to add to your patch and not add my own